docs/debug/README.md

```diff
@@ -1,3 +1,32 @@
+# How to DEBUG
+
+## DEBUG: start with process locally
+
+Start kruise-manager process locally, it can connect to a local or remote k8s-cluster.
+
+**Set up hostname**
+
+First, you should think of a fake hostname for your local machine, such as `power-machine`.
+
+Add this line into `/etc/hosts` file in which machine that runs kube-apiserver:
```

Comment:
```suggestion
Add this line into `/etc/hosts` file in dev machine that runs kube-apiserver or minikube:
```
I have tested that this changes works for dev box that runs minikube without running apiserver directly.

Reply:
Even if you configure your dev machine using minikube, then this it the machine that runs kube-apiserver.

My point is that we have not to care where the kube-apiserver runs. It may runs in local or dev machine started by minikube, or in pods of a k8s cluster, it doesn't matter.